staging/src/k8s.io/kube-aggregator/go.mod

```diff
@@ -9,11 +9,13 @@ require (
 	github.com/emicklei/go-restful/v3 v3.9.0
 	github.com/gogo/protobuf v1.3.2
 	github.com/google/gofuzz v1.2.0
+	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
 	github.com/spf13/cobra v1.6.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.1
 	golang.org/x/net v0.8.0
 	k8s.io/api v0.27.1
+	k8s.io/apiextensions-apiserver v0.27.1
```

Comment:
This shouldn't happen

Reply:
as I mentioned in https://github.com/openshift/kubernetes/pull/1654#issuecomment-1664426298, ok copying the merge functions now